spec/integration/install_spec.cr

```diff
@@ -855,7 +855,7 @@ describe "install" do
     with_shard(metadata) do
       ex = expect_raises(FailedCommand) { run "shards install --no-color" }
       ex.stdout.should contain <<-ERROR
-        E: Could not find executable "nonexistent"
+        E: Could not find executable "nonexistent" for "executable_missing"
```

Comment:
I found 2 ways to mention about dependency in Error message.

1. Just the name. Example `... for "crystal-db"`
2. With the keyword `dependency`. Example: `... for dependency "crystal-db"`

Reply:
Let's use whatever here and address standardization of message in a follow-up. Possibly with a more structured approach (https://github.com/crystal-lang/shards/pull/674#discussion_r2096071683).